rslib/backend.proto

```diff
@@ -1447,8 +1447,6 @@ message Config {
   message Bool {
     enum Key {
       BROWSER_TABLE_SHOW_NOTES_MODE = 0;
-      BROWSER_SORT_BACKWARDS = 1;
-      BROWSER_NOTE_SORT_BACKWARDS = 2;
```

Comment:
I've wondered before: Do I have to fix the numbers when reordering or removing variants? As far as I understand it, the only benefit is that it looks neater, and in other applications, where pb messages are exchanged between clients with possibly different versions, it would even be vital *not* to change the numbers.

Reply:
Currently the backend proto is considered internal, so we're free to reorder things as we please, but you're right this is something we shouldn't be doing if it were used in a network service.

If you'd like to avoid gaps without renumbering everything, one option is to move the tail elements in their place (16/17 → 1/2, sort of like https://doc.rust-lang.org/std/vec/struct.Vec.html#method.swap_remove)

Reply:
If you don't mind the gaps, I don't, either. 🙂

Reply:
They're a minor irritation, but we can always fill them when adding future entries. :-)